packages/ui-components/tsconfig.json

```diff
@@ -1,20 +1,21 @@
 {
```

Comment:
Should instead of we using tsc to compile all of this, to actually use a bundler? Such as esbuild or rollup or webpack?

Reply:
Why? We don't care about size? tsc works fine, no?

Reply:
> But maybe using a bundler wouldn't need for us to change the source?

We won't have to–I'm fixing that now